thrift/server.hh

```diff
@@ -101,9 +107,9 @@ class thrift_server {
 private:
     std::vector<server_socket> _listeners;
     std::unique_ptr<thrift_stats> _stats;
-    std::shared_ptr<::cassandra::CassandraCobSvIfFactory> _handler_factory;
+    shared_ptr_ns::shared_ptr<::cassandra::CassandraCobSvIfFactory> _handler_factory;
     std::unique_ptr<apache::thrift::protocol::TProtocolFactory> _protocol_factory;
-    std::shared_ptr<apache::thrift::async::TAsyncProcessorFactory> _processor_factory;
+    shared_ptr_ns::shared_ptr<apache::thrift::async::TAsyncProcessorFactory> _processor_factory;
     uint64_t _total_connections = 0;
```

Comment:
Please squash with the previous patch, and get rid of the unrelated patches that snuck into the branch (due to master being force-pushed, contrary to good practice).